### PR TITLE
Park data()

### LINKED ISSUE #6: Data sanitization & restructuralization
Structuring data to allow for better working of the chatbot. This means:
- using a uniform structure for every park (currently this is broken by the fact that different parks have varying numbers of POIs)
- getting rid of _extra data fields, storing info in one variable only.
- separating the different forms of public transport into their own variables (user may specify which form he prefers to use)
- uniform formatting of phone numbers (+ 420 present in some, not in others)
- adding all restaurant links (some restaurants had the link to their website in form of hyperlink, not currently stored in our data)

### PR DESCRIPTION
**Pull request by DeC0rtez**
Data collection, translation and formatting finished. Still missing Vojanovy Sady (due to reported [bug](https://github.com/DeC0rtez/SC_ChatBot/issues/5)), however this is not significant in ICA1. 

**Ready for review.**

### CODE REVIEW

File: park_info.clj [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+        :POIs "Pleasant dining in the historical town district, suitable for both children and the elderly, Church of Our Lady of the Snows"
+        ;---Children and Sport---
+        :bike_path true
+        :skating false
+        :sports_field false
+        :sports_field_extra: nil
+        :playground true
+        :playground_extra: nil
+        ;---Directions---
+        :public_transport "Wenceslaus Sq. Trams #3, #9, #14, #24, #51, #52, #54, #55, #56, #58, metro lines: A, B station: Můstek"
```
Comment (Lucius1010011010) [inline park_info.clj:45]: Maybe use a semicolon between the different transport types? As in, "...#56, #58; metro lines: A..." Make it less confusing for the end user.

File: park_info.clj [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+        :sports_field false
+        :sports_field_extra: "nearby: Youth House (www.ddmp6.cz/kontakt/) & traffic playground"
+        :playground true
+        :playground_extra: "2 playgrounds"
+        ;---Directions---
+        :public_transport "
+        stop: Nad Markétou - buses #179, #184, #191, #510, 
+        stop: Libocká - buses #179, #510, 
+        stop: Vypich - trams #15, #22, #25, #57"
+        :GPS_coords "50°5'2.798" N, 14°19'45.529" E"
```
Comment (Lucius1010011010) [inline park_info.clj]: Wrong inverted commas used, causing the string to break up.

File: park_info.clj [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+        ;---Map---
+        :map_url "https://www.praha.eu/public/bf/26/3e/184361_4_hvezda_mapa.pdf"
+
+    }
+    def Kampa{
+        ;---Basic Info---
+        :food true
+        :food_extra "restaurant: Rybí Restaurace (www.rybklub.cz),
+        café: Mlýnská (tel. +420 608 444 490)
+        restaurant/café : Sovovy Mlýny (http://www.sovovymlyny.com/)"
```
Comment (Lucius1010011010) [inline park_info.clj]: Nitpicking here, but space before the colon shouldn't be there.

File: park_info.clj [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+        :skating true
+        :sports_field false
+        :sports_field_extra: "no specialized sports facilities, but grass-covered segments available for sports"
+        :playground true
+        :playground_extra: "yes, in the vicinity: Community and Maternal centre Kampa"
+        ;---Directions---
+        :public_transport "
+        stop: Helichova - trams #12, #20, #20, #22, #23, #57 
+        stop: Újezd - trams #6, #9, #22, #23, #57, #58, #59"
+        :GPS_coords "50°5'16.87" N, 14°24'14.58" E"
```
Comment (Lucius1010011010) [inline park_info.clj]: Wrong inverted commas.

File: park_info.clj [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+        :parking false
+        :parking_extra nil
+        ;---Additional Information---
+        :dogs nil
```
Comment (Lucius1010011010) [inline park_info.clj:119]: Difference between dogs: nil and dogs: false?

File: park_info.clj [hunk 6]
```diff
@@ -0,0 +1,367 @@
...
+        :map_url "https://www.praha.eu/public/7c/a6/ed/184308_4_kampa_mapa.pdf"       
+       
+    }
+    def Kinskeho_zahrada{
+        ;---Basic Info---
+        :food true
+        :food_extra "café in Musaion museum of natural sciences (www.nm.cz)"
+        :toilet true
+        :toilet_extra nil
+        :POIs "Enjoyable location for a walk with children, Quiet and relaxing space in the center of town, Musaion Museum of Natural Sciences, decorative ponds, natural attraction - Petřín rock formations"
```
Comment (Lucius1010011010) [inline park_info.clj:133]: "Musaion Museum" - typo?

File: park_info.clj [hunk 7]
```diff
@@ -0,0 +1,367 @@
...
+        :POIs "Cultural monument, classical music concerts, social events, W.A. Mozart Museum" ;POI refers to Point of Interest
+        ;---Children and Sport---
+        :bike_path true
+        :skating false
+        :sports_field false
+        :sports_field_extra: nil
+        :playground false
+        :playground_extra: nil
+        ;---Directions---
+        :public_transport "Trams #4, #7, #9, #10, #58, #59"
```
Comment (Lucius1010011010) [inline park_info.clj]: Does not conform to standard established further (line for every stop).

File: park_info.clj [hunk 8]
```diff
@@ -0,0 +1,367 @@
...
+        ;---Children and Sport---
+        :bike_path true
+        :skating false
+        :sports_field true
+        :sports_field_extra: "Tennis courts and sports facilities in Sokolovna gymnasium"
+        :playground true
+        :playground_extra "by the garden restaurant"
+        ;---Directions---
+        :public_transport "stop: Klamovka - buses #149, #191, #217, - trams #4, #7, #9, #10, #58, #59"
+        :GPS_coords "50°4'17,04" N, 14°22'39,72" E"
```
Comment (Lucius1010011010) [inline park_info.clj]: Wrong inverted commas.

File: park_info.clj [hunk 9]
```diff
@@ -0,0 +1,367 @@
...
+        :bike_path true
+        :skating true
+        :sports_field false
+        :sports_field_extra: "Not in the park itself, but nearby: traffic playfround and Vypich Youth House"
+        :playground true
+        :playground_extra "by the restaurant and by the Ladronka bus stop"
+        ;---Directions---
+        :public_transport "
+        stop: U ladronky/ Rozýnova/ Štefkova - bus #191
+        stop: Vypich - buses #108, #174, #180, #191, #I3, - trams: #15, 22, 25, 57"
```
Comment (Lucius1010011010) [inline park_info.clj]: Either use colons before all transport types or not at all.

File: park_info.clj [hunk 10]
```diff
@@ -0,0 +1,367 @@
...
+        ;---Children and Sport---
+        :bike_path true
+        :skating true
+        :sports_field false
+        :sports_field_extra: "Not in the park itself, however sports and recreational activities are permitted on grassy segments, additionally:
+        2x skates rentals: Rental #1 - Skala sport tel. +420 605 258 670, Rental #2 - tel. +420 220 571 721
+        General purpose rental: AC Sparta rental - tel. +420 603 938 328,
+        small pitch for football, paid access tennis courts on premises of LTC Slavia - tel. +420 777 789 140"
+        :playground true
+        :playground_extra "4 in total. By the letenský estate, 2 by the tennis courts, one by Hanavský pavillion"
```
Comment (Lucius1010011010) [inline park_info.clj:230]: Include number of playgrounds in each location or not at all.

File: park_info.clj [hunk 11]
```diff
@@ -0,0 +1,367 @@
...
+        ;---Directions---
+        :public_transport "
+        stop: Letenské Náměstí/ Sparta - trams: #1, #8, #15, #25, #26, #51, #56
+        stop: Čechův most - trams: #12, #17, #53
+        stop: Chotkovy Sady - trams: #18, #20, #57
+        stop: Hradčanská metro - metro line A"
+        :GPS_coords "50°5'41" N, 14°24'57" E"
```
Comment (Lucius1010011010) [inline park_info.clj]: Wrong inverted commas.

File: park_info.clj [hunk 12]
```diff
@@ -0,0 +1,367 @@
...
+        :skating true
+        :sports_field false
+        :sports_field_extra: "Not in the park itself, but nearby: traffic playfround and Vypich Youth House"
+        :playground true
+        :playground_extra "by the restaurant and by the Ladronka bus stop"
+        ;---Directions---
+        :public_transport "
+        stop: U ladronky/ Rozýnova/ Štefkova - bus #191
+        stop: Vypich - buses #108, #174, #180, #191, #I3, - trams: #15, 22, 25, 57"
+        :GPS_coords "50°4'47.447" N, 14°21'52.445" E"
```
Comment (Lucius1010011010) [inline park_info.clj]: Wrong inverted commas.

File: park_info.clj [hunk 13]
```diff
@@ -0,0 +1,367 @@
...
+        ;---Children and Sport---
+        :bike_path true
+        :skating true
+        :sports_field true
+        :sports_field_extra: "Sokol Vinohrady"
+        :playground true
+        :playground_extra "yes, by the restaurant"
+        ;---Directions---
+        :public_transport "
+        metro: Jiřího z Poděbrad (Line A)
```
Comment (Lucius1010011010) [inline park_info.clj]: Non-standard input: "metro: info" as opposed to "stop: info".

File: park_info.clj [hunk 14]
```diff
@@ -0,0 +1,367 @@
...
+        stop: Na Smetance - bus: #135
+        stop: Italská/ Vinohradská Tržnice - tram: #11"
+        :GPS_coords "50°4'50.659" N, 14°26'26.287" E"
```
Comment (Lucius1010011010) [inline park_info.clj]: Wrong inverted commas.

File: park_info.clj [hunk 15]
```diff
@@ -0,0 +1,367 @@
...
+        :skating true
+        :sports_field true
+        :sports_field_extra: "tennis courts at TK Sparta Prague"
+        :playground true
+        :playground_extra "2 playgrounds"
+        ;---Directions---
+        :public_transport "
+        stop: Výstaviště - trams: #5, #12, #14, #15, #17, #53, #54
+        stop: Sibiřské náměstí - bus: #131
+        steamboat: RAŠÍNOVO NÁBŘEŽÍ"
```
Comment (Lucius1010011010) [inline park_info.clj]: Non-standard input.

File: park_info.clj [hunk 16]
```diff
@@ -0,0 +1,367 @@
...
+        :GPS_coords "50˚ 6' 27.80" N 14˚ 25' 12.53" E"
```
Comment (Lucius1010011010) [inline park_info.clj]: Wrong inverted commas.

File: park_info.clj [hunk 17]
```diff
@@ -0,0 +1,367 @@
...
+        ;---Children and Sport---
+        :bike_path true
+        :skating false
+        :sports_field true
+        :sports_field_extra: "tennis courts (tel. +420 606 650 717, +420 606 500 604)"
+        :playground true
+        :playground_extra "by St. Martin chapel, Mr. Trnka playpen, by entrance to Gorlice hall"
+        ;---Directions---
+        :public_transport "
+        metro: Vyšehrad (Line C)
```
Comment (Lucius1010011010) [inline park_info.clj]: Non-standard input.

File: park_info.clj [hunk 18]
```diff
@@ -0,0 +1,367 @@
...
+        stop: Podolská Vodárna - trams: #3, #16, #17, #21, #52"
+        :GPS_coords "50°3'46.66" N, 14°25'22.75" E"
```
Comment (Lucius1010011010) [inline park_info.clj]: Wrong inverted commas.

File: park_info.clj [hunk 19]
```diff
@@ -0,0 +1,367 @@
...
+        :parking true
+        :parking_extra "underground parking of Prague Congress Centre, or in Na Topolce street"
+        ;---Additional Information---
+        :dogs nil
+        :path_type "paved, threshing"
```
Comment (Lucius1010011010) [inline park_info.clj]: Threshing floor.

File: park_info.clj [hunk 20]
```diff
@@ -0,0 +1,367 @@
...
+        stop: Újezd - trams: #6, #9, #12, #20, #22, #23, #57, #58, #59
+        stop: Hellichova - trams: #12, #20, #22, #23, #57
+        stop: Pohořelec - trams: #22, #23
+        stop: Koleje Strahov - buses: #143, #149, #217"
+        :GPS_coords "50°05′2.93″ N, 14°23′40.88″ E"
+        :parking true
+        :parking_extra "by Strahov stadium"
+        ;---Additional Information---
+        :dogs nil
+        :path_type "asphalt, concrete, paved, threshing"
```
Comment (Lucius1010011010) [inline park_info.clj]: Threshing floor.

File: park_info.clj [hunk 21]
```diff
@@ -0,0 +1,367 @@
...
+        ;---Children and Sport---
+        :bike_path false
+        :skating false
+        :sports_field false
+        :sports_field_extra: nil
+        :playground true
+        :playground_extra: "yes, by the Kinských square entrance, nearby Kinských Villa and more under construction"
+        ;---Directions---
+        :public_transport "stop: Kobrova - bus #176"
+        :GPS_coords "50° 4’ 44.26’’ N, 14° 24’ 0.21’’ N"
```
Comment (Lucius1010011010) [inline park_info.clj]: Non-standard inverted commas used ('' as opposed to ″).

File: park_info.clj [hunk 22]
```diff
@@ -0,0 +1,367 @@
...
+        ;---Children and Sport---
+        :bike_path true
+        :skating false
+        :sports_field false
+        :sports_field_extra: nil
+        :playground false
+        :playground_extra: nil
+        ;---Directions---
+        :public_transport "Trams #4, #7, #9, #10, #58, #59"
+        :GPS_coords "50°4’15.00’’ N, 14°23’42.00’’ E"
```
Comment (Lucius1010011010) [inline park_info.clj]: Non-standard inverted commas used ('' as opposed to ″).

File: park_info.clj (hunk 1 above)
Comment (Lucius1010011010) [inline park_info.clj]: Non-standard input format (stop: info).

File: park_info.clj [hunk 23]
```diff
@@ -0,0 +1,367 @@
...
+        ;---Directions---
+        :public_transport "
+        stop: U ladronky/ Rozýnova/ Štefkova - bus #191
+        stop: Vypich - buses #108, #174, #180, #191, #I3, - trams: #15, 22, 25, 57"
+        :GPS_coords "50°4'47.447" N, 14°21'52.445" E"
+        :parking true
+        :parking_extra "Yes, in streets Tomanova & Skokanská"
+        ;---Additional Information---
+        :dogs true
+        :path_type "paved, threshing & asphalt"
```
Comment (Lucius1010011010) [inline park_info.clj]: Put in 'threshing floor' as opposed to 'threshing', confused the hell out of me for a bit.

File: park_info.clj [hunk 24]
```diff
@@ -0,0 +1,367 @@
...
+        :public_transport "
+        stop: Výstaviště - trams: #5, #12, #14, #15, #17, #53, #54
+        stop: Sibiřské náměstí - bus: #131
+        steamboat: RAŠÍNOVO NÁBŘEŽÍ"
+        :GPS_coords "50˚ 6' 27.80" N 14˚ 25' 12.53" E"
+        :parking true
+        :parking_extra "by the Exhibition grounds or in Bubeneč"
+        ;---Additional Information---
+        :dogs true
+        :path_type "asphalt, threshing"
```
Comment (Lucius1010011010) [inline park_info.clj]: Threshing floor.

**Review by Lucius1010011010 — CHANGES_REQUESTED** [review]
Review made, areas to be addressed identified. See comments for further details.

File: park_info.clj (hunk 5 above)
Comment (DeC0rtez) [reply park_info.clj:119]: :dogs nil = no info provided by park about dogs
:dogs false = explicitly forbidden

File: park_info.clj (hunk 6 above)
Comment (DeC0rtez) [reply park_info.clj:133]: Nope.
https://www.tripadvisor.cz/Attraction_Review-g274707-d7657382-Reviews-Musaion-Prague_Bohemia.html

File: park_info.clj (hunk 24 above)
Comment (DeC0rtez) [reply park_info.clj]: Will change.

File: park_info.clj (hunk 23 above)
Comment (DeC0rtez) [reply park_info.clj]: Will change.

File: park_info.clj (hunk 1 above)
Comment (DeC0rtez) [reply park_info.clj]: Will change.

File: park_info.clj (hunk 22 above)
Comment (DeC0rtez) [reply park_info.clj]: Common problem, pasted straight from website. Does not affect functionality to my knowledge but will fix.

File: park_info.clj (hunk 21 above)
Comment (DeC0rtez) [reply park_info.clj]: Common problem, pasted straight from website. Does not affect functionality to my knowledge but will fix.

File: park_info.clj (hunk 10 above)
Comment (DeC0rtez) [reply park_info.clj:230]: Is a textual description provided by the park, copied verbatim from [source](https://www.praha.eu/jnp/cz/co_delat_v_praze/parky/letna/index.html) (Děti a sport tab). Personally do not see this as an issue.

**Comment by DeC0rtez** [comment]
Changes made, conversations resolved except for one (number of playgrounds - see my comment).

**Review by Lucius1010011010 — APPROVED** [review]

**Pull request merged**